youtubesearchpython/core/channel.py

```diff
@@ -8,20 +8,36 @@
 from youtubesearchpython.core.componenthandler import getValue, getVideoId
 
 
-
 class ChannelCore(RequestCore):
-    def __init__(self, channelId: str):
+    result = {}
```

Comment:
It is wrong to write it here. This is global and not local. Consider the following
```python
pew = Channel('UC-lHJZR3Gqxm24_Vd_AJ5Yw', ChannelRequestType.playlists)
beet = Channel('UC2XdaAVUannpujzv32jcouQ', ChannelRequestType.playlists)
print(pew.result)
```
The expected result are playlists from pewdiepie, but the observer results are playlists from the beetles.
The correct was is to put this into the `__init__` function.

Reply:
:+1: